File: C# I/03.Operators-And-Expressions/04.Rectangles/Rectangles.cs
using System;
class Rectangles
{
    static void Main()
    {
        double width = double.Parse(Console.ReadLine());
        double height = double.Parse(Console.ReadLine());
        double perimeter = width * 2 + height * 2;
        double area = width * height;
        Console.WriteLine("{0:F2} {1:F2}", area, perimeter);
    }
}

File: JS Fundamentals/01.Operators-And-Expressions/Rectangles.js
function solve(args)
{
    let width = Number(args[0]);
    let height = Number(args[1]);
    let area = width * height;
    let perimeter = 2 * (width + height);

    console.log(area.toFixed(2) + " " + perimeter.toFixed(2));

}
solve([ '5', '5']);